Libraries/Alert/Alert.js

```diff
@@ -11,9 +11,9 @@
  */
 'use strict';
 
-const AlertIOS = require('AlertIOS');
-const NativeModules = require('NativeModules');
-const Platform = require('Platform');
+const AlertIOS = require('./AlertIOS');
+const NativeModules = require('../BatchedBridge/NativeModules');
+const Platform = require('../Utilities/Platform');
 
 import type { AlertType, AlertButtonStyle } from 'AlertIOS';
```

Comment:
Just realized that I didn't modify the `import type` statements. Not sure if this is necessary, since flow in this repo is working in `haste` mode, but maybe I should change them for both consistency as well as so that that module mode isn't required for people consuming RN with Flow.